packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -251,6 +255,20 @@ class ExpansionPanelList extends StatefulWidget {
   /// is null, then [ThemeData.dividerColor] is used.
   final Color? dividerColor;
 
+  /// The color of the arrow icon when the panel is collapsed.
+  ///
+  /// Defaults to [Colors.black54] when the theme's
+  /// [ThemeData.brightness] is [Brightness.light] and to
+  /// [Colors.white60] when it is [Brightness.dark].
+  final Color? iconColor;
+
+  /// The color of the arrow icon when the panel is expanded.
+  ///
+  /// Defaults to [Colors.black54] when the theme's
+  /// [ThemeData.brightness] is [Brightness.light] and to
+  /// [Colors.white] when it is [Brightness.dark].
```

Comment:
Looks like this code is just copied from https://master-api.flutter.dev/flutter/material/ExpandIcon/expandedColor.html? Maybe use a macro so we don't forget to update it here should it ever change.

Reply:
"Maybe use a macro so we don't forget to update it here should it ever change."
Hi, I didn't get it.

Reply:
@goderbauer any updates??

Reply:
https://pub.dev/packages/dartdoc#macros - look for examples of this in our docs.